scalafmt-core/shared/src/main/scala/org/scalafmt/util/TreeClasses.scala

```diff
@@ -17,3 +16,14 @@ object CtorModifier {
   def unapply(tree: Tree): Boolean =
     tree.is[Mod.Private] || tree.is[Mod.Protected]
 }
+@classifier
+trait CaseClass
```

Comment:
I find it's a bit ad-hoc to limit this to case classes, why not primary constructors for all `class`? We could apply it only to Ctor.Primary